src/trix/controllers/editor_controller.coffee

```diff
@@ -33,7 +33,7 @@ class Trix.EditorController extends Trix.Controller
     @editor = new Trix.Editor @composition, @selectionManager, @editorElement
     if document?
       @editor.loadDocument(document)
-    else
+    else if html isnt ''
```

Comment:
Creating a new, empty document should take no time at all. Unless there's an issue, I don't see a good reason for adding an additional condition around it.